src/schemas/components/SchemaFormProperty.vue

```diff
@@ -64,7 +88,7 @@
       return null
     },
     set(value) {
-      emit('update:value', value)
+      emit('update:value', value ?? undefined)
```

Comment:
Our core prefect-design components generally emit `null` for empty values. So making sure we coerce `null` values to `undefined` here.